### DISCUSSION TITLE
Installation from scratch on Mac OSX M1 doesn't work due to outdated dependencies

### DISCUSSION THREAD

**Discussion by claytongulick** [apache/superset#18329] [Q&A / Help]
Currently installing via pip install apache-superset fails because the version of outdated dependencies with grpcio, numpy, pandas and pyarrow that are referenced in setup.py won't build on arm64.

#### How to reproduce the bug

1. Follow installation instructions on a Mac with a M1 processor

### Expected results

The installation should succeed

### Actual results

Several errors occur building dependency packages

**Comment by nutellaweera** [comment]
I'm running into the same issue, can't go down the docker installation path either due to https://github.com/apache/superset/issues/14104. 
@claytongulick did you find a workaround by any chance? Or is superset simply not supported for m1 users at the moment?

**Reply by claytongulick** [reply]
No, I've got my head in the sand with this one. Doing other work and hoping it gets fixed before I move back to implementing superset (or something else for BI). Don't really want to be forced to use WSL on my Wintendo just for superset :-(

**Comment by mayurnewase** [comment]
Try installing in Rosetta mode, and then run in normal mode.
always works for me.

**Comment by genzgd** [comment]
Also facing this issue.  If there's not active work in this area maybe Preset could take responsibility?  M1 Macs are not going to be a niche platform for the sorts of developers who want to kick the tires on superset, and trying to install in Rosetta mode is not an obvious or completely straightforward process.
